adafruit_dash_display.py

```diff
@@ -125,18 +125,23 @@ def last_val(self, value: str):
 
 
 class Hub:  # pylint: disable=too-many-instance-attributes
-    """Object that lets you make an IOT dashboard"""
+    """
+    Object that lets you make an IOT dashboard
+
+    :param display: The display object.
+    :param io: The io object.
+    :param nav: The navigation buttons.
```

Comment:
The parameters have type annotations, so we should include matching type annotations in the parameter documentation as well. Also made the descriptions clearer.
```suggestion
    :param displayio.Display display: The display for the dashboard.
    :param IO_MQTT io_mqtt: MQTT communications object.
    :param Tuple[DigitalInOut, ...] nav: The navigation pushbuttons.
```

Reply:
Good call! I wouldn't have known what to put for the first two. Thank you so much for the code suggestion!